build.gradle

```diff
@@ -1,12 +1,49 @@
 apply plugin: 'java'
 apply plugin: 'eclipse'
 apply plugin: 'idea'
+apply plugin: 'maven'
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
 repositories {
     mavenCentral()
 }
 
- dependencies {
-        testCompile (
-            "junit:junit:4.11",
-        )
+install {
+    repositories.mavenInstaller {
+        pom.version = '0.0.1-SNAPSHOT'
+        pom.artifactId = 'gdpr-consent-framework'
+        pom.groupId = 'com.yieldlab'
+    }
+}
+
+uploadArchives {
+    repositories {
+        mavenDeployer {
+            repository(url: "file://~/.m2/")
+            pom.version = '0.0.1-SNAPSHOT'
+            pom.artifactId = 'gdpr-consent-framework'
+            pom.groupId = 'com.yieldlab'
```

Comment:
If we want to contribute back, this should not be `com.yieldlab`, but I guess for the IAB this is not set up at all - so we'll fix that later if needed.

Reply:
i can set it back to com.iab if we wish to contribute to IAB